locations/spiders/flaming_grill_gb.py

```
@@ -0,0 +1,26 @@
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class FlamingGrillGBSpider(SitemapSpider, StructuredDataSpider):
+    name = "flaming_grill_gb"
+    item_attributes = {
+        "brand": "Flaming Grill",
+        "brand_wikidata": "",
+    }
+    allowed_domains = ["flamingrillpubs.co.uk"]
+    sitemap_urls = ["https://www.flaminggrillpubs.co.uk/sitemap.xml"]
+    sitemap_rules = [(r"\/pubs\/([-\w]+)\/([-\w]+)\/?$", "parse_sd")]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)